packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -116,9 +120,13 @@ export const participantListSelector: ParticipantListSelector = createSelector(
       // Local participant can never remove themselves.
       isRemovable: false
     });
+    /* @conditional-compile-remove(total-participant-count) */
+    const totalParticipantCount = partitipantCount;
```

Comment:
question on this -- does this include participants that are in lobby or earlier states that we typically hide from the participant list?